trace_context/HTTP_HEADER_FORMAT.md

```diff
@@ -144,12 +149,37 @@ boolean traced = (traceFlags & FLAG_TRACED) == FLAG_TRACED
 
 #### Requested Flag (00000001)
```

Comment:
I think this should be renamed to sampled as that's far more common usage

Reply:
"sampled" as a name has a confusion. If "sampled"  means "sampled in" or "sampled out"? The name `requested` eliminates this confusion.

`requested` creates another confusion. Is it `debug` or `sampled`. As specification does not define how sampling probability or sampling rate can be propagated - this confusion is easier to describe in text.